en/xml/fo/topic.graphic-pageheader.xml

```diff
@@ -0,0 +1,231 @@
+                            <para>Here you can insert a text like <emphasis>This
+                                    page is intentionally left
+                                blank</emphasis>.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.center">
+                            <para>This is a good place for the chapter
+                                title.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.right">
+                            <para>This is the place for our logo as described at
+                                the top of our solution.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.height">
+                            <para>To avoid conflicts with the body text area, we
+                                limit its height to 1 inch. When you specify
+                                only one parameter like 
+                                    <tag>height</tag> or <tag>width</tag>, your
+                                logo will be scaled with the <link
+                                    xlink:href="#li.fo.external-graphics"
+                                    >correct aspect ratio</link>.</para>
+                        </callout>
+                        <callout
+                            arearefs="co.topic.graphic-pageheader.attribute">
+                            <para>The file name is passed on the <tag
+                                    class="attribute">src</tag> attribute.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.template">
+                            <para>The template which is called to import and
+                                place the logo.</para>
+                        </callout>
+                        <callout arearefs="co.topic.graphic-pageheader.file">
+                            <para>Our file defined at the top of our
+                                customization layer.</para>
+                        </callout>
+                    </calloutlist>
+            </step>
+            <step>
+                <para>Use your customization layer to transform your DocBook
+                    document into FO.</para>
+            </step>
+        </procedure>
+    </section>
+    <section role="discussion">
+        <title>Discussion</title>
+        <para>In line 8 of our style sheet (
+            <filename>graphic-pageheader.xsl</filename>), we defined a new
+            parameter to represent the path for our logo. The main goal was to
+            be independent of the global image source path 
+             <tag>img.src.path</tag> parameter for pictures, graphics and so 
+            on. This new parameter is used in line 45 to build the path
+             for our logo.</para>
```

Comment:
Actually, this is written more from an implementation point of view. If you change to the user's view, it could like this:

```
The stylesheet <filename>graphic-pageheader.xsl</filename> creates a logo in the right edge of your page
and uses the parameter <parameter>logo.src.path</parameter> to find the logo. By default, a logo is stored
in <filename>logos/<filename>. If your logos are in a different location, change it with your the XSLT processor.
For example, the following line uses ...
```